cxx-qt-lib/src/qsizef.rs

```diff
@@ -63,3 +63,9 @@ unsafe impl ExternType for QSizeF {
     type Id = type_id!("QSizeF");
     type Kind = cxx::kind::Trivial;
 }
+
+impl From<&QSizeF> for QSizeF {
```

Comment:
This is a good point, currently this is used in the generated code to convert from a CppObj to Data. It uses the same `.into()` for all types (as opaque types into makes more sense).

I wonder if we combine this change with the generation changes in #9 as it will be a similar area of code.

And we should also consider the implications of this not using the new() method and creating the object on the C++ side. As the user might consider using clone() themselves.

Reply:
If we did do clone() i wonder if instead of using `*qsizef` if it should instead do `QSizeF::new(qsizef.w, qsize.h);` As this then creates the object on C++ side.

Then for invokables they should take `&QSizeF` as a ref (and through our wrapper) to avoid a copy. This will then be like C++ where you do `const QSizeF&` vs `QSizeF`.

Reply:
Reported #58

Reply:
I agree that this can be tracked by a separate issue :+1: